src/shared_modules/rsync/testtool/agentEmulator.cpp

```diff
@@ -33,27 +33,11 @@ static DBSYNC_HANDLE createDbsyncHandle(const std::string& dbName)
     return handle;
 }
 
-struct SmartDeleterJson final
-{
-    void operator()(cJSON* data)
-    {
-        cJSON_Delete(data);
-    }
-};
-
-struct CJsonDeleter final
-{
-    void operator()(char* json)
-    {
-        cJSON_free(json);
-    }
-};
-
 static void callback(const ReturnTypeCallback /*type*/,
                      const cJSON* /*json*/,
                      void* /*ctx*/)
 {
-    // const std::unique_ptr<char, CJsonDeleter> spJsonBytes{ cJSON_PrintUnformatted(json) };
+    // const std::unique_ptr<char, CJsonSmartFree> spJsonBytes{ cJSON_PrintUnformatted(json) };
```

Comment:
What is this callback function used for?

Reply:
this callback is for debugging only.